src/ResourceManager/Network/Commands.Network/help/Get-AzureRmApplicationGatewayBackendHealth.md

```diff
@@ -37,6 +37,8 @@ The Get-AzureRmApplicationGatewayBackendHealth cmdlet gets application gateway b
 
 
 
+
+
```

Comment:
@EvgenyAgafonchikov same comment about removing the extra blank lines in this file